includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -470,60 +475,61 @@ public function get_settings(): WP_REST_Response {
+				'is_multi_currency_enabled'            => WC_Payments_Features::is_customer_multi_currency_enabled(),
+				'is_wcpay_subscriptions_enabled'       => WC_Payments_Features::is_wcpay_subscriptions_enabled(),
+				'is_stripe_billing_enabled'            => WC_Payments_Features::is_stripe_billing_enabled(),
+				'is_wcpay_subscriptions_eligible'      => WC_Payments_Features::is_wcpay_subscriptions_eligible(),
+				'is_subscriptions_plugin_active'       => $this->wcpay_gateway->is_subscriptions_plugin_active(),
+				'account_country'                      => $this->wcpay_gateway->get_option( 'account_country' ),
+				'account_statement_descriptor'         => $this->wcpay_gateway->get_option( 'account_statement_descriptor' ),
+				'account_statement_descriptor_kanji'   => $this->wcpay_gateway->get_option( 'account_statement_descriptor_kanji' ),
+				'account_statement_descriptor_kana'    => $this->wcpay_gateway->get_option( 'account_statement_descriptor_kana' ),
+				'account_business_name'                => $this->wcpay_gateway->get_option( 'account_business_name' ),
+				'account_business_url'                 => $this->wcpay_gateway->get_option( 'account_business_url' ),
+				'account_business_support_address'     => $this->wcpay_gateway->get_option( 'account_business_support_address' ),
+				'account_business_support_email'       => $this->wcpay_gateway->get_option( 'account_business_support_email' ),
+				'account_business_support_phone'       => $this->wcpay_gateway->get_option( 'account_business_support_phone' ),
+				'account_branding_logo'                => $this->wcpay_gateway->get_option( 'account_branding_logo' ),
+				'account_branding_icon'                => $this->wcpay_gateway->get_option( 'account_branding_icon' ),
+				'account_branding_primary_color'       => $this->wcpay_gateway->get_option( 'account_branding_primary_color' ),
+				'account_branding_secondary_color'     => $this->wcpay_gateway->get_option( 'account_branding_secondary_color' ),
+				'account_domestic_currency'            => $this->wcpay_gateway->get_option( 'account_domestic_currency' ),
+				'is_payment_request_enabled'           => 'yes' === $this->wcpay_gateway->get_option( 'payment_request' ),
+				'is_debug_log_enabled'                 => 'yes' === $this->wcpay_gateway->get_option( 'enable_logging' ),
+				'payment_request_enabled_locations'    => $this->wcpay_gateway->get_option( 'payment_request_button_locations' ),
+				'payment_request_button_size'          => $this->wcpay_gateway->get_option( 'payment_request_button_size' ),
+				'payment_request_button_type'          => $this->wcpay_gateway->get_option( 'payment_request_button_type' ),
+				'payment_request_button_theme'         => $this->wcpay_gateway->get_option( 'payment_request_button_theme' ),
+				'payment_request_button_border_radius' => WC_Payments_Features::is_stripe_ece_enabled() ? $this->wcpay_gateway->get_option( 'payment_request_button_border_radius' ) : 10,
```

Comment:
When ECE is disabled that implies that the Payment Request Buttons are enabled instead. Those do not support changing the border radius, so we need a fallback value when ECE is disabled.

A default radius of 10px was something I just chose arbitrarily while working on the PR. We'll probably want some other default radius for when ECE is disabled, likely to match what the PRBs use.

Reply:
The default radius [should be 4px](https://github.com/Automattic/woocommerce-payments/pull/9010#issuecomment-2200514880).

Reply:
Fixed in https://github.com/Automattic/woocommerce-payments/pull/9010/commits/b76d492b54b2137d1193677bfc225d9e5dd0658e